tests/frontend/unit/Filter.spec.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * (c) 2010-present DEMOS E-Partizipation GmbH.
+ *
+ * This file is part of the package demosplan,
+ * for more information see the license file.
+ *
+ * All rights reserved
+ */
+
```

Comment:
Shouldn't this file be moved to the `store` subdirectory?

Reply:
right. Moved this and some others 
d05e308de6f395621bc62016363a60924f3e099c